docs/js/python.md

```diff
@@ -1,21 +1,49 @@
-# Python Support
+# Python in MakeCode
 
-**Python support in MakeCode is experimental**
+Our Python support in MakeCode is called Static Python (SPY). As with TypeScript,
```

Comment:
We decided to call it just MakeCode Python instead of Static Python. Lets omit this line and refer to the python implementation as MakeCode Python.